### DISCUSSION TITLE
MacOS Monterey Compatibility

### DISCUSSION THREAD

**Discussion by BeansOnToastMJR** [Whisky-App/Whisky#767] [Q&A]
I'm totally new to this and not very tech savvy.. With Whisky requiring MacOS 14 and higher, is there a previous release for MacOS 12 I can download? If not, any advice on how I can run Lethal Company on MacOS 12?

**Comment by sdlotu** [comment]
Whisky requires the GPTK, which only functions with the Apple Silicon, or M series CPUs. So there will never be a version for any earlier MacOS versions. It won't even run under Sonoma on an Intel system, as it's not the OS, but the hardware that makes Whisky possible with GPTK.

It is possible you can run it with Crossover from CodeWeavers, or using the PortingKIt from www.paulthetall.com.
